Automattic/jetpack
# Protect: Add dedicated manual WAF rules section

#### Description
Adds a dedicated section for the manual rules settings

#### Changes proposed in this Pull Request:

* Modifies the structure of the primary section to display conditionally based on state settings
* Adds a `setWafisUpdating` action to replace the `setWafIsLoading` action
* Uses `isUpdating` for all WAF settings updates, and isLoading for only the primary WAF toggle
* Introduces a section beneath the manual rules toggle section that allows for the display of conditional content (no rules message, and eventually stats) and an edit rules button
* Adds foundational actions for the post-upgrade messaging
* Fixes all failing stories
* Fixes mobile styling issues in the `FirewallFooter` component when the module is disabled

#### Other information:


#### Jetpack product discussion


- 1201069996155224-as-1203286905507152

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:






* Checkout this branch
* Load your Jurassic Tube
* Activate Protect
* Proceed to Protect admin page and select the Firewall tab
* Enable the Firewall
* Enable Manual rules
* Select Edit manual rules
* Verify that a new manual rules settings section is loaded in place of the primary section
* Toggle Go back and verify that the primary section is reloaded
* Ensure no regression in app functionality
